roles/cifmw_cephadm/defaults/main.yml

```diff
@@ -91,8 +91,8 @@ cifmw_ceph_rgw_config:
 cifmw_cephadm_cephfs_name: "cephfs"
 cifmw_cephadm_ceph_spec_fqdn: "{{ ceph_spec_fqdn | default(false) | bool }}"
 cifmw_ceph_dashboard_spec_path: /tmp/ceph_dashboard.yml
-cifmw_cephadm_certificate: ""
-cifmw_cephadm_key: ""
+cifmw_cephadm_certificate: "/etc/pki/tls/example.com.crt"
+cifmw_cephadm_key: "/etc/pki/tls/example.com.key"
```

Comment:
When I use https://github.com/openstack-k8s-operators/ci-framework/pull/1865 with the above two defaults I see the certificate created like this:
```
[zuul@controller-0 ci-framework]$ oc get secret cert-ceph-rgw -o yaml | grep alt-names
    cert-manager.io/alt-names: compute-0.ctlplane.example.com,compute-1.ctlplane.example.com,compute-2.ctlplane.example.com
[zuul@controller-0 ci-framework]$ 
```
which fit's what's on the host:
```
[root@compute-0 ~]# hostname -f
compute-0.ctlplane.example.com
[root@compute-0 ~]# 
```
These are the endpoints:
```
sh-5.1$ openstack endpoint list | egrep "swift|manila-public"
| 3c146221e4254ddc924a748912a529a7 | regionOne | swift        | object-store | True    | public    | http://192.168.122.2:8080/swift/v1/AUTH_%(tenant_id)s                    |
| a78f2a1361f84c81989b7709bf59001f | regionOne | manila       | share        | True    | public    | https://manila-public-openstack.apps.ocp.openstack.lab/v1/%(project_id)s |
| d7fa9d3ea3f34d91817e70d9fb7baee4 | regionOne | manilav2     | sharev2      | True    | public    | https://manila-public-openstack.apps.ocp.openstack.lab/v2                |
| f3413c55f72b43e0996430f7b66d8caf | regionOne | swift        | object-store | True    | internal  | http://192.168.122.2:8080/swift/v1/AUTH_%(tenant_id)s                    |
sh-5.1$ 
```
I included manila above just to show the name `https://manila-public-openstack.apps.ocp.openstack.lab` though `example.com` seems to go with what the EDPM host gets from ci-framework.

So this shouldn't be a problem. It's just defaulted to our opinionated testing environment.